running-recipes/getting-started.md

```diff
@@ -335,6 +337,34 @@ dependencies {
 }
 ```
 {% endcode %}
+{% endtab %}
+
+{% tab title="Kotlin" %}
+{% code title="build.gradle.kts" %}
+plugins {
+    `java-library`
+    `maven-publish`
+    id("org.openrewrite.rewrite") version "6.1.19"
```

Comment:
Surprised this line didn't need to be changed in Kotlin, but away from my laptop so can't check right this instant. Calling it out for myself later.

Reply:
For the groovy setup it's ` id 'org.openrewrite.rewrite' version '6.1.19'`